app/src/main/java/com/iic/mokojin/CurrentMatchFragment.java

```diff
@@ -50,34 +54,30 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         View rootView = inflater.inflate(R.layout.fragment_current_match, container, false);
         ButterKnife.inject(this, rootView);
+        mEventBus = ((Application)getActivity().getApplication()).getCurrentSessionStore().getEventBus();
```

Comment:
this is a pretty nasty line and you use it multiple times.
how about 
`SessionStore.getEvenBus(getActivity)`